src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20200318093829_UpdatingAbsenceKeyStatDataBlocks.Designer.cs

```diff
@@ -0,0 +1,4086 @@
tr>\\n    <tr>\\n      <td>Absence rates</td>\\n      <td></td>\\n      <td>Overall, authorised and unauthorised absence rate breakdowns</td>\\n    </tr>\\n    <tr>\\n      <td>Those who are persistent absentees</td>\\n      <td></td>\\n      <td>\\n        Those missing 10 per cent or more of their own possible sessions in the\\n        period\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Absence by reason</td>\\n      <td></td>\\n      <td>A breakdown of absence sessions by reason</td>\\n    </tr>\\n    <tr>\\n      <td>PRU absence</td>\\n      <td></td>\\n      <td>\\n        Overall, authorised and unauthorised absence rates, persistent absentee\\n        rates and absence by reason in pupil referral units (PRUs)\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Pupils who miss one or more session due to absence by reason</td>\\n      <td></td>\\n      <td>\\n        The number of pupils missing one or more session for overall, authorised\\n        and unauthorised absence and for individual reasons for absence\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Absence and persistent absence by pupil characteristic</td>\\n      <td>* ~</td>\\n      <td>\\n        Overall, authorised, unauthorised and persistent absence rates and\\n        absence by reason by gender, free school meal eligibility, free school\\n        meal eligibility in the last 6 years, national curriculum year group,\\n        first language, SEN provision and ethnic group.\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Pupil absence in schools by IDACI decile</td>\\n      <td>*</td>\\n      <td>\\n        Including overall, authorised and unauthorised absence rates and\\n        persistent absence rates by income deprivation affecting children\\n        indices (IDACI) decile of pupil residence and school location.\\n        [Available at LA District level]\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Four year old absence</td>\\n      <td></td>\\n      <td>Overall absence rates for four year olds</td>\\n    </tr>\\n    <tr>\\n      <th colspan=\\\"3\\\">School level, available via underlying data only</th>\\n    </tr>\\n    <tr>\\n      <td>Absence rates</td>\\n      <td></td>\\n      <td>Overall, authorised and unauthorised absence rate breakdowns</td>\\n    </tr>\\n    <tr>\\n      <td>Those who are persistent absentees</td>\\n      <td></td>\\n      <td>\\n        Those missing 10 per cent or more of their own possible sessions in the\\n        period\\n      </td>\\n    </tr>\\n    <tr>\\n      <td>Absence by reason</td>\\n      <td></td>\\n      <td>A breakdown of absence sessions by reason</td>\\n    </tr>\\n    <tr>\\n      <td>Four year old absence</td>\\n      <td></td>\\n      <td>Overall absence rates for four year olds</td>\\n    </tr>\\n  </tbody>\\n</table>\\n\",\"Type\":\"HtmlBlock\",\"Id\":\"7cc516d4-fc79-4e22-b35b-a042d5b14d35\",\"Order\":0,\"Comments\":null}],\"Release\":null},{\"Id\":\"97a138bf-4ebb-4b17-86ab-ed78584608e3\",\"Order\":5,\"Heading\
```

Comment:
I'm not expecting all these content changes and this might re-introduce the markdown display issue with spacing in the bulleted lists that we had a few months ago. Check what a fresh migration from master creates.